scripts/ci/verify_modified_index.sh

```diff
@@ -26,17 +28,19 @@ while read line; do
     echo
     echo "New index entries detected for extension:" $ext
     echo "Adding latest entry from source:" $source
-    set +e
+
     az extension add -s $source -y
-    if [ $? != 0 ]; then
-        continue
-    fi
-    set -e
-    echo "Load all commands"
-    azdev verify load-all
-    echo "Running linter"
-    azdev cli-lint --ci --extensions $ext
+
+    # TODO migrate to public azdev
+    echo "Load all commands..."
+#    azdev verify load-all
+
+    # TODO migrate to public azdev
+    echo "Running linter..."
+#    azdev cli-lint --ci --extensions $ext
```

Comment:
should we run the azdev linter and azdev verify load-all?

Reply:
Currently, the public [azdev](https://github.com/Azure/azure-cli-dev-tools) doesn't have those two commands. So it's a dangerous hotfix which will open the Pandora Box that leave extension commands without some correctness verification.